packages/esm-service-queues-app/src/hooks/useSystemSetting.ts

```diff
@@ -14,5 +14,8 @@ export function useSystemSetting(setting: string) {
     systemSetting: data?.data,
     error: error,
     isLoading: isLoading,
+    isValueUuid:
```

Comment:
Yes, actually the RefApp has the concept Name in the system setting value and KGH PIH server has conceptUUID.

Currently, it's the only place it's needed, so I added it here only instead of a helper function, and it is something which a developer should take care of, similar to what I have done in `useServiceQueues`

Reply:
This function just fetches data from the server. I think we should leave interpreting that data (in this case, whether it is a uuid) to the caller, even if we don't separate that out as a helper function.